### DISCUSSION TITLE
Configure for HTTPS

### DISCUSSION THREAD

**Discussion by bsouza-site** [netbox-community/netbox#10343] [General]
Hello everyone

I'm trying to get Netbox to work in HTTPS, I followed the information from the official Netbox page, however, when I try to access the page via HTTPs, the browser returns the message: **ERR_SSL_PROTOCOL_ERROR.**

I checked with systemctl status netbox, the service has the message: **You're accessing the development server over HTTPS, but it only supports HTTP**

Has anyone experienced this and know how to fix this error?

**Comment by jsenecal** [comment]
ERR_SSL_PROTOCOL_ERROR is thrown when you are accessing a (plain text) HTTP endpoint from a browser that expects (SSL/TLS socket) HTTPS.
This means that something must not have been configured right.

**Reply by bsouza-site** [reply]
Thanks for the information jsenecal!

Could you tell me how to change the development server to HTTPS?

**Reply by jeremystretch** [reply]
> You're accessing the development server over HTTPS, but it only supports HTTP

The development server does not support HTTPS because it is not suitable for production use. Please follow the [official installation guide](https://docs.netbox.dev/en/stable/installation/) to ensure NetBox is installed correctly.

**Reply by jsenecal** [reply]
Ohhh ***development server***, yeah, no, that will never work, @jeremystretch is right.

**Comment by bsouza-site** [comment]
Sorry for the lack of answers and thank you for the information!

**Comment by bsouza-site** [comment]
Many thanks to everyone for the information.

I configured a new Netbox following the documentation and it is working correctly under https.
